### DISCUSSION TITLE
Errors in the Filament app.css and class build warning

### DISCUSSION THREAD

**Discussion by chester-af** [filamentphp/filament#14536] [Help] [answered]
### Package

Panel builder

### Package Version

V3.2.117

## app.css errors
I recently updated Filament from `v3.2.92` to `V3.2.117` and now there seems to be a lot of errors in the Filament app.css.
Path to the file: `public/css/filament/filament/app.css`

Here is an image of some of the errors:
<img width="500" alt="filament-app css-errors" src="https://github.com/user-attachments/assets/b52edc0b-4c15-40d8-9a27-2feadf31cf1c">

Everything seems to work okay but I'm just a little concerned/curious as to why this file now has so many errors?

## Filament class build warning
When I run `npm run build` or `npm run dev` and click around the panel I see this warning in the console:
`warn - The utility disabled:[-webkit-text-fill-color:theme(colors.gray.500)] contains an invalid theme value and was not generated.`

I've searched my project for this tailwind class and I only found it in a few Filament components.
Here is a path to a component with this class:
`vendor/filament/forms/resources/views/components/date-time-picker.blade.php`

I downgraded the project's Filament back to `V3.2.117` just to see if I still get the console warning and strangely I do so I must have missed it when I was last working on the project. 

I created a fresh Laravel project with Filament and added a input that has that class to see if I got the warning but I didn't.

Is there a way to fix this? I'm assuming if I'm not getting the warning on the fresh install of Laravel that it must be something I've done to cause this error but like I stated I only found the class in Filament components.

Many thanks for any help/advice on this!

**Comment by CushtyDev** [comment]
Did you manage to fix this issue? Thanks

**Comment by fraschne2** [comment]
I am having the exactly the same issue. Noticed it only today when updating to filament v3.2.125.

**Reply by CushtyDev** [reply]
Any fix for this issue? Thanks

**Comment by Daniil-Solovyev** [comment]
Same problem. Any solutions?

**Comment by chester-af** [comment]
Hi, sorry guys, I never found a solution to this.

@danharrin sorry to disturb you, but do you know what we could do to fix this? I can create a bug issue if you think it is a bug? Thanks!

**Reply by danharrin** [reply]
I don't think its a bug, looks like the IDE/build tool just doesn't understand the Tailwind. These are warnings and not really errors right?

**Reply by chester-af** [reply]
Thanks for the reply. PHPStorm thinks they are errors, there are also warnings:
<img width="853" alt="image" src="https://github.com/user-attachments/assets/97b3867d-bc1e-407d-b51d-a1c49398a876" />

I've got two projects, one is just a regular Laravel website (using tailwind) and the other is a Filament site with a custom theme, if I run `npm run build` on each of them, and go look in the compiled css file, I can only see errors in the Filament site and if I remove Filament paths from my `tailwind.config.js` then the errors also go away which made me think it could be a bug, but maybe PHPStorm just can't find figure it out because its coming from a package?

**Reply by danharrin** [reply, marked as answer]
Maybe since there is less magic in Tailwind 4 there won't be as many problems. But these errors are not accurate. The missing CSS variables get generated at runtime.
